src/Canvas.tsx

```diff
@@ -371,12 +383,30 @@ const InternalCanvas: FC<CanvasProps & { ref?: Ref<CanvasRef> }> = forwardRef(
           width={canvasWidth}
           onClick={onCanvasClick}
         >
-          {arrow !== null && (
+          {arrow !== null && arrowHover !== null && (
             <defs>
-              <CloneElement<MarkerArrowProps>
-                element={arrow}
-                {...(arrow as MarkerArrowProps)}
-              />
+              <React.Fragment key={`${id}-arrow`}>
```

Comment:
Let's make it just `Fragment` like:

```suggestion
              <Fragment key={`${id}-arrow`}>
```